subsys/partition_manager/pm.yml.zboss

```diff
@@ -0,0 +1,9 @@
+#include <autoconf.h>
+
+zboss_nvram:
+  placement: {after: [mcuboot_storage, app]}
```

Comment:
I assume that this should be page-aligned? If so consider adding an `align` property as well.

```suggestion
  placement: {after: [app], align: {start: DT_FLASH_ERASE_BLOCK_SIZE}}
```

Reply:
Good point.